internal/backup/volumesnapshotbackup_action.go

```diff
@@ -58,5 +65,40 @@ func (p *VolumeSnapshotBackupBackupItemAction) Execute(item runtime.Unstructured
 		return nil, nil, errors.WithStack(err)
 	}
 
-	return &unstructured.Unstructured{Object: vsbMap}, nil, nil
+	// Add temp VSClass as additional item
+	tempVSC := snapshotv1api.VolumeSnapshotClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: fmt.Sprintf("%s-snapclass", backup.Name),
+			Labels: map[string]string{
+				util.WaitVolumeSnapshotBackup: "true",
+			},
+		},
+		// dummy driver as it is not used, but the field is required
+		Driver:         "foo",
```

Comment:
This works on 4.11? If yes then we're good to go.

Reply:
tested with success on 4.11